package.json

```diff
@@ -9,14 +9,15 @@
   },
   "scripts": {
     "react-dev": "webpack -d --watch",
-    "server": "nodemon ./server/server.js",
-    "seed": "node seed/imageSeeder.js",
+    "server": "nodemon -r dotenv/config ./server/server.js",
+    "seed": "node -r dotenv/config ./seed/imageSeeder.js",
```

Comment:
are you going to add another script to run the server with just node as opposed to nodemon for deployment? example: "start" : "node -r dotenv/config ./server/server.js" or something like that?

Reply:
I'll have to inject the variable on the ec2 instance that will be needed. ^^this command pulls the variable from my .env file and injects it before nodemon runs. Once the script is injected in the environment I think pm2 will handle running the file instead of this script